main/reference/zoe-api/zoe.md

```diff
@@ -3,300 +3,333 @@
 any object.
 
-Returns a `publicFacet` containing the public facet defined for `instance`.
 ```js
-const ticketSalesPublicFacet = await E(zoe).getPublicFacet(sellItemsInstance);
+const ticketSalesPublicFacet = await E(Zoe).getPublicFacet(sellItemsInstance);
 ```
-## `E(zoe).getInvitationIssuer()`
-- Returns `{Issuer}`
 
-Zoe has a single `invitationIssuer` for its entire
-lifetime. By having a reference to Zoe, a user can get the `invitationIssuer`. This lets them claim any
-invitation they receive from someone else by calling `E(invitationIssuer).claim()` with the
-untrusted invitation as the argument. During the claiming process, the `invitationIssuer` validates
-the invitation.
+## E(Zoe).getInvitationIssuer()
+- Returns: **Promise&lt;[InvitationIssuer](./zoe-data-types.md#invitationissuer)>**
 
-The `mint` associated with the `invitationIssuer`
-creates `invitations` in the form of ERTP `payments` that represent the right to interact with
-a smart contract in particular ways.
-
-The `invitationIssuer` has two methods, both of which take an `invitation` as an argument.
-Remember, an `invitation` is just a special case of an ERTP `payment`, so `claim()` and
-`getAmountOf()` are the same as for other `issuers`.
-
-A successful call of `invitationIssuer.claim(invitation)` means you are assured the `invitation`
-is recognized as valid by the `invitationIssuer`. You are also assured the `invitation` is exclusively yours
-and no one else has access to it.
+Returns a **Promise** for the **InvitationIssuer** for the Zoe instance.
 
 ```js
-const invitationIssuer = await E(zoe).getInvitationIssuer();
+const invitationIssuer = await E(Zoe).getInvitationIssuer();
 // Here a user, Bob, has received an untrusted invitation from Alice.
-// Bob uses the trusted `invitationIssuer` from Zoe to
+// Bob uses the trusted **InvitationIssuer** from Zoe to
 // transform the untrusted invitation to a trusted one
 const invitation = await invitationIssuer.claim(untrustedInvitation);
 const { value: invitationValue } =
     await E(invitationIssuer).getAmountOf(invitation);
 ```
 
-## `E(zoe).getInvitationDetails(invitation)`
-- `invitation` `{Invitation}`
-- Returns `{Promise<Object>}`
+## E(Zoe).getInvitationDetails(invitation)
+- **invitation** **Invitation**
+- Returns **Promise&lt;Object>**
+
+Takes an **Invitation** as an argument and returns a **Promise** for an object containing the following
+details about the **Invitation**:
 
-Takes an `invitation` as an argument and returns an object containing the following
-details about the `invitation`:
-- `installation` `{Installation}`: The contract's installation in Zoe.
-- `instance` `{Instance}`: The contract instance this invitation is for.
-- `invitationHandle` `{Handle}`: A handle used to refer to this invitation.
-- `description` `{String}`: describes the purpose of this `invitation`. Use it
+- **installation** **Installation**: The contract's Zoe installation.
```

Comment:
For previous examples, the fields of the return object are shown in a code block after an example of making a call to the API. This should be done consistently.